Contact/Contact Class and Smaller Related Data Classes/Contact.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Nate.ContactApp
+{
+    public class Contact
+    {
+        // Ideally need a 2-wide list of attributes with only a couple default attributes (ie first/last name)
+        public int ContactID;
+        public bool IsActiveRecord;
+        public string FirstName;
+        public string LastName;
+        public PhoneNumber Home;
+        public PhoneNumber Work;
+        public PhoneNumber Cell;
+        public EmailAddress Email;
+        public PhysicalAddress Address;
+        public string Notes;
+
+        public Contact()
+        {
+            ContactID = 0;
+            IsActiveRecord = true;
+            FirstName = "";
+            LastName = "";
+            Home = new PhoneNumber();
+            Work = new PhoneNumber();
+            Cell = new PhoneNumber();
+            Email = new EmailAddress();
+            Address = new PhysicalAddress();
+            Notes = "";
+        }
+
+        public Contact(int newID)
```

Comment:
I'm not sure I get your use case in which you would only want to set the id.  I'd either make everything settable via constructor or just use this class as a plain old data model and let consumers set the fields they need to.

Reply:
You're right. I'll be deleting that. It was something I had started and never had any real plan for.

Reply:
Deleted.